packages/terra-date-picker/src/react-datepicker/stylesheets/react_datepicker.module.scss

```diff
@@ -487,8 +477,11 @@
     }
   }
 
-  .is-calendar-focused {
-    outline: var(--terra-date-picker-month-focus-outline, 2px dashed #000);
-    outline-offset: var(--terra-date-picker-month-focus-outline-offset, 0);
+  .is-calendar-focused,
+  .react-datepicker-day {
+    &--keyboard-focus {
+      outline: var(--terra-date-picker-day-focus-outline, 2px dashed #000);
+      outline-offset: var(--terra-date-picker-day-month-focus-outline-offset, 0);
```

Comment:
These styles should also be applies to the selected day (the `.react-datepicker-day--selected` class).

Reply:
Applied here 251400f

Reply:
We would probably need to flex this for the selected day. We do want to be able to apply the dashed outline to the selected day when it is focused but the styles should be removed once it focus is lost. Currently, if there is a selected day, the focus style is always there and doesn't get removed when the focus is moved to a different day.

![Screen Shot 2020-03-02 at 2 56 07 PM](https://user-images.githubusercontent.com/22618655/75717092-16777a00-5c96-11ea-86bc-33e6fc61a95b.png)

Reply:
@saket2403 I'm still seeing the selected day keeping the dashed border when moving to a different preselected day.